src/mailroom.py

```diff
@@ -0,0 +1,169 @@
+# -*- coding: utf-8 -*-
+""" This is the mailroom madness script. It will run from the cmd line and
+ allow you to generate an email or report based on user input."""
+
+import os
+import sys
+import math
+
+
+DONOR_DICT = {'Linus Torvalds': [100, 150, 250],
+              'Jane Goodall': [200, 250, 50],
+              'Bob Ross': [1000, 550, 1]}
+
+
+def main_menu_input():
+    '''Display user prompt for main menu'''
+    user_input = input('''
+    Main Menu:
+    E to generate a thank you email to send to a donor
+    R for a report on the donation history of all donors
+    Type exit at any prompt to exit the program.
+
+    >>>''').upper()
+    menu_input_validation(user_input)
```

Comment:
Here, if we returned the value input, rather than sending it directly into the input validation function, we could isolate requesting user input from the rest of the program.  That would be cool, no?